specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -1850,6 +1850,536 @@
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "cacheName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the redis cache.",
+            "pattern": ".*"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of list access policies.",
+            "schema": {
+              "$ref": "#/definitions/RedisCacheAccessPolicyListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/redis/{cacheName}/accessPolicyAssignments/{accessPolicyName}": {
+      "put": {
+        "tags": [
+          "Redis",
+          "AAD"
+        ],
+        "operationId": "AccessPolicyAssignment_CreateUpdate",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "RedisCacheAccessPolicyAssignmentCreateUpdate": {
+            "$ref": "./examples/RedisCacheAccessPolicyAssignmentCreateUpdate.json"
+          }
+        },
+        "description": "Adds the access policy assignment to the specified users",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "cacheName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache.",
+            "pattern": ".*"
+          },
+          {
+            "name": "accessPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the access policy to assign.",
+            "pattern": "^[a-zA-Z0-9]+$"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RedisCacheAccessPolicyAssignmentCreateUpdateParameters"
```

Comment:
>RedisCacheAccessPolicyAssignmentCreateUpdateParameters [](http://example.com/codeflow?start=37&length=54)

Should merge definitions of RedisCacheAccessPolicyAssignmentCreateUpdateParameters and RedisCacheAccessPolicyAssignmentResponse into
`RedisCacheAccessPolicyAssignment`

Reply:
Merged RedisCacheAccessPolicyAssignmentResponse and RedisCacheAccessPolicyAssignments